pkg/tests/ossm/yaml_configs.go

```diff
@@ -31,6 +31,12 @@ var (
 
 	//go:embed yaml/smmr.yaml
 	smmr string
+
+	//go:embed yaml/multiple-namespaces.yaml
+	multiple_namespaces string
+
+	//go:embed yaml/multiple-test-smmr.yaml
+	multiple_smmr string
```

Comment:
"multiple_smmr" may not be a good name. It actually is one smmr with multiple members.
I suggest we call it "smmr_multiple_members"